charts/zigbee2mqtt/Chart.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: v2
-name: zigbee2mqtt
-type: application
-description: Bridges events and allows you to control your Zigbee devices via MQTT
 appVersion: 1.15.0
-version: 2.1.0
+description: Bridges events and allows you to control your Zigbee devices via MQTT
+name: zigbee2mqtt
+version: 3.0.0
 keywords:
   - zigbee
   - mqtt
+  - home-assistant
```

Comment:
Maybe replace home-assistant with home-automation? It's not really that tightly coupled with HA.

Reply:
i think home-assistant is fine, it's related and in the zwave chart too :)